### PR TITLE
Math magician test

### PR DESCRIPTION
**Pull request by gergitokusse**
## 👇 👇 The following feature and file were added to the branch 👇 👇 
👉 Unit test for each project component with a file named `component.test.js`
👉 Unit test for operate.js and calculate.js

### CODE REVIEW

File: src/test/component.test.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  });
+  test('Test AC btn on calculatore', () => {
+    render(<Mycalculator />);
+    const pElement = screen.getByText(/AC/i);
+    expect(pElement).toMatchSnapshot();
+  });
+
+  test('Test % btn on calculatore', () => {
+    render(<Mycalculator />);
+    const pElement = screen.getByText(/%/i);
```
Comment (Zuhaib042) [inline src/test/component.test.js]: - [ ] All your tests are in this file on point but there is a small issue. What component test meant in the requirement is that you create different files for different components. for example for your component `Calculatore.js` create a test file names `calculatore.test.js`, for home component, create a test file named `home.test.js` and so on and so fourth. 
by convention we don't test all our components in one file, we actually create different test files for our different components. kindly consider implement this change in organizing your test files.

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @gergitokusse 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✅ Descriptive PR
✅ Professioanl Readme

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/test/component.test.js (hunk 1 above)
Comment (gergitokusse) [reply src/test/component.test.js]: 🙏🙏🙏 Thank you very much for your good and constructive comment. we will do it according your comment.

**Review by rustam-tolipov — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Developer-512px-10

## Highlights
1. No Linters errors 💯
2. All requirements are complete 💯
3. Good readme 👍 
4. Well-seperated test files 👍 

## Optional suggestions
No optional suggestions :)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**